rep-2005.rst

```diff
@@ -0,0 +1,245 @@
+
+* Orchestration
+
+  * `ros2/launch <https://github.com/ros2/launch>`_
+  * `ros2/launch_ros <https://github.com/ros2/launch_ros>`_
+
+* Features
+
+  * `ros/diagnostics <https://github.com/ros/diagnostics>`_
+  * `ros/xacro <https://github.com/ros/xacro>`_
+  * `ros/robot_state_publisher <https://github.com/ros/robot_state_publisher>`_
+  * Sensor processing
+
+    * `ros-perception/image_common <https://github.com/ros-perception/image_common>`_
+    * `ros-perception/image_transport_plugins <https://github.com/ros-perception/image_transport_plugins>`_
+    * `ros-perception/perception_pcl <https://github.com/ros-perception/perception_pcl>`_
+    * `ros-perception/vision_opencv <https://github.com/ros-perception/vision_opencv>`_
+    * `ros-perception/laser_filters <https://github.com/ros-perception/laser_filters>`_
+    * `ros-perception/laser_geometry <https://github.com/ros-perception/laser_geometry>`_
+    * `ros-perception/laser_proc <https://github.com/ros-perception/laser_proc>`_
+    * `ros-perception/depthimage_to_laserscan <https://github.com/ros-perception/depthimage_to_laserscan>`_
+
+  * `ros-planning/navigation2 <https://github.com/ros-planning/navigation2>`_
+  * `ros-planning/moveit2 <https://github.com/ros-planning/moveit2>`_
+  * `ros-visualization/interactive_markers <https://github.com/ros-visualization/interactive_markers>`_
+  * `ros2/geometry2 <https://github.com/ros2/geometry2>`_
+  * `ros2/kdl_parser <https://github.com/ros2/kdl_parser>`_
+  * `ros2/message_filters <https://github.com/ros2/message_filters>`_
+  * `ros2/sros2 <https://github.com/ros2/sros2>`_
+  * `ros2/teleop_twist_joy <https://github.com/ros2/teleop_twist_joy>`_
+  * `ubuntu-robotics/ament_nodl <https://github.com/ubuntu-robotics/ament_nodl>`_
+  * `ubuntu-robotics/nodl <https://github.com/ubuntu-robotics/nodl>`_
+  * ROS Drivers
+
+    * `ros2/joystick_drivers <https://github.com/ros2/joystick_drivers>`_
+    * `ros-drivers/velodyne <https://github.com/ros-drivers/velodyne>`_
+    * `ros-drivers/urg_c <https://github.com/ros-drivers/urg_c>`_
+    * `ros-drivers/urg_node <https://github.com/ros-drivers/urg_node>`_
+    * `ros-drivers/phidgets_drivers <https://github.com/ros-drivers/phidgets_drivers>`_
+    * `KumarRobotics/imu_vn_100 <https://github.com/KumarRobotics/imu_vn_100>`_
+
+* Documentation, Examples, Tutorials
+
+  * `ros2/demos <https://github.com/ros2/demos>`_
+  * `ros2/design <https://github.com/ros2/design>`_
+  * `ros2/examples <https://github.com/ros2/examples>`_
+  * `ros2/ros_core_documentation <https://github.com/ros2/ros_core_documentation>`_
+  * `ros2/ros2_documentation <https://github.com/ros2/ros2_documentation>`_
+
+* Robot
+
+  * `ros/urdfdom_headers <https://github.com/ros/urdfdom_headers>`_
+  * `ros2/urdf <https://github.com/ros2/urdf>`_
+  * `ros2/urdfdom <https://github.com/ros2/urdfdom>`_
+
+* Tools
+
+  * `ros-simulation/gazebo_ros_pkgs <https://github.com/ros-simulation/gazebo_ros_pkgs>`_
```

Comment:
Webots is being increasingly used by ROS2 users and Cyberbotics is committed to maintain the integration of Webots in ROS2 on the long term. This will be possible thanks to the support of several EU research projects including an upcoming [ROSin FTP](https://rosin-project.eu/ftps) aiming at improving the ROS2 support in Webots and [OpenDR](https://opendr.eu/) that includes the use of ROS2 for Webots simulations. The [Webots ROS2 Wiki page](http://wiki.ros.org/webots_ros2) describes the current status of the integration which is fully functional and demonstrated in this [ROS2 simulation of UR robots](http://wiki.ros.org/webots_ros2/Tutorials/webots_ros2_universal_robot).

```suggestion
  * `ros-simulation/gazebo_ros_pkgs <https://github.com/ros-simulation/gazebo_ros_pkgs>`_
  * `cyberbotics/webots_ros2 <https://github.com/cyberbotics/webots_ros2>`_
```

Reply:
In addition to that, I'd also recommend adding packages that integrate [Ignition](https://ignitionrobotics.org) (i.e. Gazebo's next generation) to ROS 2. There are several ROS 2 + Ignition demos documented [here](https://github.com/osrf/ros_ign/tree/dashing/ros_ign_gazebo_demos).

```suggestion
  * `ros-simulation/gazebo_ros_pkgs <https://github.com/ros-simulation/gazebo_ros_pkgs>`_
  * `osrf/ros_ign <https://github.com/osrf/ros_ign/>`_
```